introduction.md

```diff
@@ -0,0 +1 @@
+We'll need to know your GitHub username for this course. We'll assume all changes for this course are done by a GitHub user with this username
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.